### ISSUE TITLE
 Lower recreated accuracy of bert-base on SNLI with Human-in-loop data augmentation than your paper shown.

### ISSUE THREAD

**Issue by Siki-cloud** [thu-coai/AutoCAD#2]
Hi, thank you for recommending me your job; it truely has been of great reference to me.

But i still have some question about the recreated accuracy with the Human-CAD SNLI task.  I used the follwing dataset but got lower more accuracies of snli, mnli-m,mnli-mm than your experiment results. 

   -  The sample numbers of  Human-CAD all combed SNLI dataset provided by (Kaushik et al., 2019) are around 8k, 1.6k, 0.8k  in train/test/dev dataset repestively.
   -  mine:  40.195	38.013	39.0862  VS  yours:  85.75    66.26    66.14
   -  some hyperparameters during fine-tuning bert-base-uncased as below:
            -       batch size，lr=16，1e-5
            -      decay，optimizer = 0 , adam
            -      early stopping = 5
            -      seed = 42
  
   I attempted to align the parameters as described in your paper and try different seeds, but still failed to touch your accuracies.   I have a few doubts:

   - What does "human-cad" refer to in your paper?
   - Which datasets (dev/test/train) were used to measure the accuracy of MNLI-m and MNLI-mm?

 Please help me.

**Comment by Jiaxin-Wen** [comment]
1. For the NLI task, human-cad refers to the dataset provided by Kaushik
2. The official test set of MNLI-m and MNLI-mm

**Comment by Siki-cloud** [comment]
I found the point that i uncorrectly mapped the lable:str -> id: int, which "contradiction" in ood is 2, but when i training, i mapped "contradiction" into 1.  
Thank you for your respond.  

btw,  may i ask whether  human-cad  was supplemented as same number of examples as the original SNLI dataset  20,000 examples? 

I noticed that the article mentioned this statement :
---"Human-CAD The Human-CAD dataset (Kaushik et al., 2019) is created by employing human annotators to rewrite a subset of the SNLI dataset. In our experiment, we ensure that our training dataset includes all the samples in the subset."

 so I'm a bit confused about the size of the HUD dataset used in the experiment?  Thank you.
